test/test_playbooks/repository.yml

```diff
@@ -79,6 +105,25 @@
         vars:
           repository_state: present
           repository_label: "just_a_test_repo"
+          repository_mirror_on_sync: false
+          repository_name: Test Docker Repository
+          repository_content_type: docker
+          repository_url: https://docker.io
+          repository_docker_upstream_name: busybox
+          expected_change: true
+      - include: tasks/repository.yml
+        vars:
+          repository_state: present
+          repository_mirror_on_sync: false
+          repository_name: Test Docker Repository
+          repository_content_type: docker
+          repository_url: https://docker.io
+          repository_docker_upstream_name: busybox
+          expected_change: false
+      - include: tasks/repository.yml
+        vars:
+          repository_state: present
+          repository_mirror_on_sync: true
```

Comment:
https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-info.html#L116-L123 suggest that mirror on sync cannot be set for docker repos.

Intresting that is seems to report something has changed... Maybe a server bug

Reply:
I'll remove the additional tests from the docker repository tests, to be honest I didn't properly read (or manually verify if its even possible via the webui) https://projects.theforeman.org/issues/21305.